esp/platform/txsummary.cpp

```diff
@@ -90,6 +139,11 @@ void CTxSummary::serialize(StringBuffer& buffer) const
         else
             buffer.appendf("%s;", entry.key.str());
     }
+
+    for (const std::pair<StringBuffer, CumulativeTimer*>& entry : m_timers)
+    {
+        buffer.appendf("%s=%" I64F "ums;", entry.first.str(), entry.second->getTotalMillis());
```

Comment:
This is a bit strange, but correct. The format token for an unsigned 64 bit integer differs between Linux ("%llu") and Windows ("%I64u"). The syntax in question allows the preprocessor to assemble the correct form of the format string.